content/events/hacktoberfest.adoc

```diff
@@ -37,14 +37,20 @@ See the link:/participate/[Contribute and Participate] page for more information
 1. Sign-up to Hacktoberfest on link:https://hacktoberfest.digitalocean.com[the event website].
 2. Join link:https://gitter.im/jenkinsci/hacktoberfest[our Gitter channel].
 3. Everything is set, just start creating pull-requests!
+** This year Hacktoberfest does require labeling pull requests,
```

Comment:
Match with the later statement that Hacktoberfest does not require labeling pull requests.

```suggestion
** This year Hacktoberfest does not require labeling pull requests,
```